service-front/Gruntfile.js

```diff
@@ -60,15 +60,18 @@ module.exports = function (grunt) {
       }
     },
 
-    // govuk template css copied into public/assets
+    // govuk template css - copy our fork of the template to public dir, as we
```

Comment:
comment that we will not need this later when we are using the CSS directly from the node module